indico/modules/events/registration/templates/management/_registration_details.html

```diff
@@ -25,7 +25,9 @@
         {% if registration.state.name not in ('rejected', 'withdrawn') %}
             {{ _render_withdraw_actions(registration) }}
         {% endif %}
-
+        <div class="action-box">
+            {{ _render_registration_tags(registration, assigned_tags, all_tags) }}
+        </div>
```

Comment:
```suggestion
        {% if all_tags %}
            <div class="action-box">
                {{ _render_registration_tags(registration, assigned_tags, all_tags) }}
            </div>
        {% endif %}
```

No need to show the tagging UI if no tags are defined